File: CSharp/SpaceVIL/SpaceVIL/Cores/EventTask.cs
using System;
using System.Threading;
using System.Collections.Generic;
using System.Collections.Concurrent;

namespace SpaceVIL
{
    internal sealed class EventTask
    {
        public InputEventType Action = 0;
        public Prototype Item = null;
        public InputEventArgs Args = null;
    }
}

File: spacevil_21052019/java_src/EventTask.java
package com.spvessel.spacevil;

import com.spvessel.spacevil.Core.InputEventArgs;
import com.spvessel.spacevil.Flags.InputEventType;

final class EventTask {
    InputEventType action = InputEventType.EMPTY;
    Prototype item = null;
    InputEventArgs args = null;
}